frontend-ssr/components/cli_git/GitCli.module.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/variables';
+
+.commands {
+    width: 35vw;
+
+    .commands__cli {
+        height: 50vh;
+        overflow: auto;
+        padding: 2vh 1vw;
```

Comment:
On mobile devices this padding might not be big enough. Let's use `em` from now on regarding sizes. If you've never used it, read about it from MDN.

Reply:
I've used `em` in my previous projects. Thanks!